test/model.test.js

```diff
@@ -0,0 +1,11 @@
+const itemsCreationModel = require('../src/model')
+const assert = require('assert').strict;
```

Comment:
From my understanding there are benefits using import instead of require whenever possible, because: https://insights.untapt.com/webpack-import-require-and-you-3fd7f5ea93c0

Do you agree? If yes is it possible to use import here? I also asked about require vs. import in the discord from the Odin Project. Let's see what they answer. ;)

Reply:
We need to find a way to use import statement. I have tried it with import statement but got an error.